File: Programmer_test/PT_Python/P12940.py
def solution(n, m):
    answer = [1, 1]
    mi = min(n, m)
    ma = max(n, m)
    for i in range(1, mi+1):
        if ((m % i == 0) and (n % i ==0)):
            answer[0] = i
    for i in range(ma, mi * ma + 1):
        if ((i % m == 0) and (i % n == 0)):
            answer[1] = i
            break
    return answer

print(solution(2, 5))

File: Programmer_test/PT_Java/P12940.java
public class P12940 {
    public static void main(String[] args) {
        P12940 T = new P12940();
        System.out.println(T.solution(5, 2));    
    }

    public int[] solution(int n, int m) {
        int[] answer = {1, 1};
        int min = n;
        int max = m;
        if (min > m) {
            min = m;
            max = n;
        }
        for (int i = 2; i <= min; i++) {
            if ((n % i == 0) && (m % i == 0)) {
                answer[0] = i;
            }
        }
        for (int i = min; i <= max * min; i++) {
            if ((i % n == 0) && (i % m == 0)) {
                answer[1] = i;
                break;
            }
        }
        return answer;
    }
}
